ores/scoring_systems/celery_queue.py

```diff
@@ -34,7 +35,8 @@ def update_sent_state(sender=None, body=None, **kwargs):
 
 class CeleryQueue(ScoringSystem):
 
-    def __init__(self, *args, application, queue_maxsize=None, **kwargs):
+    def __init__(self, *args, application, queue_maxsize=None,
+                 jobs_lock_manager=None, **kwargs):
```

Comment:
"jobs_lock_manager" seems to be both too specific and not closely enough related to celery_queue.  There's no notion of a "job" in celery.  There are tasks and results.  I think this might be better named as a "task tracker" or "request deduplicator" or something like that.  Related: https://blog.codinghorror.com/i-shall-call-it-somethingmanager/

Reply:
I renamed it and moved it to task_tracker (and redis_task_tracker)